lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -2090,6 +2091,11 @@ instance Arbitrary (PostTransactionOldData n) where
         <*> arbitrary
         <*> arbitrary
 
+instance Arbitrary TxMetadataWithSchema where 
+  arbitrary = TxMetadataWithSchema 
+    <$> elements [TxMetadataNoSchema, TxMetadataDetailedSchema]
+    <*> arbitrary
```

Comment:
Having a shrinker here would help with debugging the failures. I think just using `generic{Arbitrary, Shrink}` is fine for this wrapper
```haskell 
instance Arbitrary TxMetadataSchema where
    arbitrary = genericArbitrary
    shrink = genericShrink

instance Arbitrary TxMetadataWithSchema where
  arbitrary = genericArbitrary
  shrink = genericShrink
```

Before: `* Value matches more than one of 'oneOf' schemas: Object (fromList [("string",String "\1105689")])`
After `* Value matches more than one of 'oneOf' schemas: Object (fromList [("string",String "a")])`

Reply:
Thanks, adding it